<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QWidget" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>云南高速公路车道维护软件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnMin">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnMax">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>■</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnClose">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="WidgetTool" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="tb_Home">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>直接启动</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_Update">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>软件升级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_Setting">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>车道配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_devtest">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>外设测试</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_File">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>文件传输</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_Log">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>日志查看</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_DataResend">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>数据重传</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
